### ISSUE TITLE
redoc-cli 0.8.1 tries to install styled-components 4.1.12 which does not exist

### ISSUE THREAD

**Issue by Quaiks** [Rebilly/ReDoc#848]
Hello!

A couple of hours ago all our pipelines are failing when executing `npx redoc-cli ...` with the following error.

> npm ERR! code ETARGET
npm ERR! notarget No matching version found for styled-components@^4.1.12
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'redoc-cli'
npm ERR! notarget

The last version published of `styled-components` is 4.1.3

Regards!

**Comment by martincognite** [comment]
It seems to have broken `redoc-cli < 0.8.1` as well, here's an example with `0.7.0`.
## Dockerfile
```
FROM node:10

RUN yarn global add redoc-cli@0.7.0

ENTRYPOINT [ "redoc-cli" ]
```

## build & run
```
❯ docker build -t redoc-cli .
Sending build context to Docker daemon  2.048kB
Step 1/3 : FROM node:10
 ---> 8fc2110c6978
Step 2/3 : RUN yarn global add redoc-cli@0.7.0
 ---> Running in 480b47dd9410
yarn global v1.13.0
[1/4] Resolving packages...
warning redoc-cli > styled-components > memoize-one@4.1.0: New custom equality api does not play well with all equality helpers. Please use v5.x
[2/4] Fetching packages...
info fsevents@1.2.7: The platform "linux" is incompatible with this module.
info "fsevents@1.2.7" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning "redoc-cli > redoc > swagger2openapi > better-ajv-errors@0.6.1" has unmet peer dependency "ajv@4.11.8 - 6".
[4/4] Building fresh packages...
success Installed "redoc-cli@0.7.0" with binaries:
      - redoc-cli
Done in 12.52s.
Removing intermediate container 480b47dd9410
 ---> c2d2e6088bb0
Step 3/3 : ENTRYPOINT [ "redoc-cli" ]
 ---> Running in b407fad87176
Removing intermediate container b407fad87176
 ---> 3574c01dfff8
Successfully built 3574c01dfff8
Successfully tagged redoc-cli:latest

/tmp/rcli 21s
❯ docker run --rm redoc-cli
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'base64-js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:6145:18)
    at __webpack_require__ (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:36:30)
    at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:1113:14)
    at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:2893:30)
    at __webpack_require__ (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:36:30)
    at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/redoc/bundles/redoc.lib.js:6139:41)
```

**Comment by skwashd** [comment]
I've raised a pull request to fix this. See #849.

**Comment by cyril1929** [comment]
Please accept the merge request promptly.

**Comment by RomanGotsiy** [comment]
Fixed `redoc-cli@0.8.2`.

> A couple of hours ago all our pipelines are failing when executing `npx redoc-cli`

To avoid this in the future pinpoint to a specific `redoc-cli` version (`npx redoc-cli@0.x.x bundle ...`) and update manually.

Sorry for inconveniences 🙈.
